src/Components/App/App.js

```diff
@@ -76,8 +83,19 @@ class App extends Component {
           />
         </Route>
 
+        <Route 
+          exact 
+          path='/form'>
+            <Form 
+            activities={this.state.activities} 
+            setActivities={this.setActivities}
+            updateActivityAnswers={this.updateActivityAnswers}
+            />
```

Comment:
My branch has `determineRandomActivities` being passed as a prop here, just a note for if we run into conflicts